app/services/handle_submission_for_store.rb

```diff
@@ -208,4 +217,38 @@ def school
   def user
     @user ||= store["current_user"]
   end
+
+  def uk_country
+    @uk_country ||= ISO3166::Country.find_country_by_any_name("United Kingdom")
+  end
+
+  def teacher_catchment_country
+    return uk_country.iso_short_name if in_uk_catchement_area?
+
+    store["teacher_catchment_country"]
+  end
+
+  def teacher_catchment
+    store["teacher_catchment"]
+  end
+
+  def in_uk_catchement_area?
+    teacher_catchment.in?(Application::UK_CATCHMENT_AREA)
+  end
+
+  def teacher_catchment_iso_country_code
```

Comment:
could we add testing for the different values we could get, similar to the catchment country?